docs/js/files/modules_FormModel.js.html

```diff
@@ -161,6 +161,12 @@ <h1 class="file-heading">File: modules/FormModel.js</h1>
     mapping: undefined,
 
     /**
+     * @property models
+     * @type Object
+     **/
+    models: undefined,
```

Comment:
What are our thoughts about existing methods relating to "models"? Like `getTrackedModel`, `getTrackedModels` or `setTrackedModel` or `setTrackedModels`.

I guess the question is two-fold: Will people be tempted in changing your new `this.models` directly hoping it calls these methods under the hood? And should methods like `setMapping` and `setMappings` default to use `this.models`?